test-app/src/main/java/org/readium/r2/testapp/reader/ReaderActivity.kt

```diff
@@ -38,20 +39,20 @@ import org.readium.r2.testapp.outline.OutlineFragment
  */
 open class ReaderActivity : AppCompatActivity() {
 
-    private lateinit var modelFactory: ReaderViewModel.Factory
-    private lateinit var model: ReaderViewModel
+    private val model: ReaderViewModel by viewModels()
+
+    override fun getDefaultViewModelProviderFactory(): ViewModelProvider.Factory {
```

Comment:
This change seems pointless but I refactored it twice. At some point `TtsViewModel` was a proper Android `ViewModel` and so I had a composite view model factory.